### DISCUSSION TITLE
Purchasing aid required

### DISCUSSION THREAD

**Discussion by ledermann** [raomin/ESPAltherma#434] [Q&A] [answered]
Hey folks! Since my Daikin Altherma 3 H MT ECH2O (`ETSH12P50EF`) is not performing very well, I would like to monitor it more closely. I stumbled upon your project, which looks great, but since I am not a hardware nerd, I would like to ask for advice on what to buy.

Please, can someone check if this is the right stick?
https://www.amazon.de/M5StickC-PLUS2-Development-Entwicklungsset-Generation/dp/B0CWFN391J

And, perhaps more importantly, what is the right cable to connect to my Daikin heat pump?

**Comment by philipgo** [comment, marked as answer]
Yes, the M5Stickc Plus 2 is suitable, although the price in your link is too high. Check e.g. Eckstein or Berrybase.

**Reply by ledermann** [reply]
Thanks a lot, I just placed the order :)

**Reply by ledermann** [reply]
Just received the M5StickC Plus 2. Great - the installation of ESPAltherma was quite easy, WLAN and MQTT are working fine. The only thing I'm struggling with is the cable to connect to my Daikin, see #435.
